.github/workflows/e2e-run-on-main.yml

```diff
@@ -1,5 +1,6 @@
 name: Run safe-react e2e tests
 on:
+  pull_request:
```

Comment:
Why on each PR? There's another job that does that. This one was meant only for main.

Reply:
What is the preferred way to trigger the e2e tests against the `safe-react` PR?

Reply:
You already correctly modified [the right yml](https://github.com/gnosis/safe-react-e2e-tests/pull/92/commits/c681ca2c7b2a2e147f3019f4a9ca67c6ae6067d9#diff-dfb4b4e7044018e0f28b838985f41c03464a524c093ca95e1abf987a61151fcdR22-R23).
It will run the tests on this PR, pointing to 3015 in safe-react.